livedata/src/test/java/io/getstream/chat/android/livedata/PaginationTest.kt

```diff
@@ -18,7 +19,9 @@ class PaginationTest {
         pagination: AnyChannelPaginationRequest,
         expectedList: List<ChannelEntityPair>
     ) {
-        inputList.applyPagination(pagination) `should be equal to` expectedList
+        // show an easy to use diff between the two results
```

Comment:
Check only the `cid` doesn't cover all cases. It is assuming our `cid` will be unique, but it is something that we can't guarantee with our own models.
Apart of that, test should guarantee the full list content is the one we expect

Reply:
`cid` is a unique identifier for channels